ex01.exs

```diff
@@ -74,14 +75,26 @@ end
 defmodule Ex01 do
 
   def counter(value \\ 0) do
-    # ...your code
+    receive do
+        {:next, sent} -> send sent, {:next_is, value} 
+            counter(1+value) 
+    end
```

Comment:
This layout is confusing: it's not immediately clear that the `send` and the recursive call to `counter` are related. Also, the variable name `sent` doesn't really mean anything.  Perhaps:

~~~elixir
    receive do
        {:next, from} -> 
            send from, {:next_is, value} 
            counter(1+value) 
    end
~~~

# -1